<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>926</width>
    <height>701</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="BlackAWhite">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>100</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Black A White</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>100</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Flip</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Crop">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>150</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Crop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Resize">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>150</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Resize</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Load">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>10</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Save">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>10</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Image</string>
    </property>
   </widget>
   <widget class="QLabel" name="display_before">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>231</width>
      <height>241</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="display_after">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>210</y>
      <width>231</width>
      <height>241</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Vertical">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>530</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Vertical</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Horezintal">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>530</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Herozintal</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="StartPoints">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>500</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="Diemnsions">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>500</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labeltext">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>470</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>start points</string>
    </property>
   </widget>
   <widget class="QLabel" name="labeltext2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>470</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string> Diemnsions</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Apply">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>540</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Apply Crop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Clear">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>10</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="purple">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>200</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>purple</string>
    </property>
   </widget>
   <widget class="QPushButton" name="inferared">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>200</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>inferared</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Demon">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>250</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Demon</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rand">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>250</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Zompie</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Grayscale">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>300</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Gray Scale</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DetectEdges">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>300</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Detect Edges</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Skew">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>350</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Skew</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DarkenALighten">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>350</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>darken_or_lighten</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>460</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>180</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Before</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>179</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>After</string>
    </property>
   </widget>
   <widget class="QLabel" name="slaydervalue">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>460</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="filtername">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>490</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Filter name</string>
    </property>
   </widget>
   <widget class="QPushButton" name="filtername_2">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>490</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>opthion</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Blur">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>400</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Blur</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Apply_2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>490</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>apply</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Invert">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>400</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Invert</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Blur_2">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>450</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Rotate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="third">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>530</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>thied</string>
    </property>
   </widget>
   <widget class="QPushButton" name="wanou">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>450</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>wanou</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addframe">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>500</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Frame</string>
    </property>
   </widget>
   <widget class="QPushButton" name="red">
    <property name="geometry">
     <rect>
      <x>612</x>
      <y>610</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>red</string>
    </property>
   </widget>
   <widget class="QPushButton" name="black">
    <property name="geometry">
     <rect>
      <x>502</x>
      <y>610</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>black</string>
    </property>
   </widget>
   <widget class="QPushButton" name="blue">
    <property name="geometry">
     <rect>
      <x>392</x>
      <y>610</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>blue</string>
    </property>
   </widget>
   <widget class="QPushButton" name="green">
    <property name="geometry">
     <rect>
      <x>722</x>
      <y>610</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>green</string>
    </property>
   </widget>
   <widget class="QPushButton" name="oilpainting">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>500</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Oil Painting</string>
    </property>
   </widget>
   <widget class="QPushButton" name="oldtv">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>550</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>old tv</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Merge">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>550</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Merge</string>
    </property>
   </widget>
   <widget class="QLabel" name="omar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>630</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>OMAR BETAWY</string>
    </property>
   </widget>
   <widget class="QLabel" name="awad">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>630</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>OMAR AWAD</string>
    </property>
   </widget>
   <widget class="QLabel" name="amr">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>630</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>AMR KHALED</string>
    </property>
   </widget>
   <widget class="QLabel" name="sahla">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Sahla Bas Sa7la</string>
    </property>
   </widget>
   <widget class="QPushButton" name="undo">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>220</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Undo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Redo">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>290</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Redo</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>926</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
